list-doi-requests/src/main/java/no/unit/nva/doirequest/list/ListDoiRequestsHandler.java

```diff
@@ -114,17 +114,17 @@ private boolean userHasRequestedRole(String role, String userRolesCsv) {
         return addDoiRequestMessagesToDoiRequests(doiRequests);
     }
 
+    private Publication[] fetchDoiRequestsForCurator(UserInstance userInstance) {
```

Comment:
re-arranged methods for the method  `addDoiRequestMessagesToDoiRequests` to be under the two methods calling it and not in between